src/http.h

```diff
@@ -35,10 +35,12 @@ class HttpStateData : public Client
         /// stores the corresponding decision
         Answers make(const Answers ans, const char *why);
 
-        Answers answer; ///< the decision id
-        const char *reason; ///< the decision reason
-        const StoreEntry *entry; ///< entry for debugging
-        const Http::StatusCode statusCode; ///< HTTP status for debugging
+        Answers answer = reuseNot; ///< the decision id
+        const char *reason = nullptr; ///< the decision reason
+        const StoreEntry *entry = nullptr; ///< entry for debugging
+        const Http::StatusCode statusCode = scNone; ///< HTTP status for debugging
+    private:
+        ReuseDecision();
```

Comment:
Can we delete the unused constructor instead of hiding it?

Reply:
done